internal/filters/filters.go

```diff
@@ -42,7 +44,7 @@ func WithAllowedMethod(method string, next http.Handler) http.Handler {
 		if r.Method != method {
 			w.Header().Set("Content-Type", "application/json")
 			w.WriteHeader(http.StatusMethodNotAllowed)
-			w.Write([]byte(`{"code":405,"message":"method not allowed"}`))
+			w.Write([]byte(`{"code":405,"message":"method not allowed"}`)) //nolint:errcheck,gosec
```

Comment:
Why is the linter disabled here, but in `WithAuthentication` an error handling is implemented?

Reply:
Ah, I missed that this one is also in the filters package. I was wondering if https://github.com/gardener/oidc-webhook-authenticator/pull/163/files/897dc09fd9173cba7ddd6812c00033e3c7b44774#diff-0043929b176a6dd563ab4f5b393f095138a1200b3a22647ca5aa8e112579c80a should log the error, but here I think we can use the controller-runtime logger

Reply:
Yes, please use the controller runtime logger.